source/docs/version/index.html.slim

```diff
@@ -15,7 +15,8 @@
             = version
             small &nbsp;▽
       ol.docs-nav__list.dropdown-list data-dropdown-list="versions"
-        - versions.reverse_each do |version|
+        -  sorted = versions.sort { |a, b| b.to_s <=> a.to_s }
```

Comment:
Could we potentially move this into a helper method in `ApplicationHelper`? Perhaps `sorted_versions`?

Reply:
From my quick look through, I think this is the only place where we need the functionality. We could move it into a helper method, but I don't think its necessary 🍕

Reply:
I think there are still benefits to extracting a helper method here. Specifically to avoid the logic within the view around sorting and keeping the view "dumb" so to speak.

Reply:
+1 to using an application helper; helps keep the template nice and clean. Thanks for this, @JasonEtco!

Reply:
Ok, sounds good to me 👍 I'll make it happen. I don't write a lot of Ruby so I'll fumble through it until it works.